### ISSUE TITLE
Adjust hapi-swagger config of route doc

### ISSUE THREAD

**Issue by T1m1** [premily/bemily#8]
Adjust config to create a body in PUT and POST like:
http://petstore.swagger.io/#!/user/updateUser
or
http://petstore.swagger.io/#!/pet/updatePet

**Comment by timoweiss** [comment]
Is this what you were looking for:
https://github.com/premily/bemily-authentication/blob/master/src/plugin.ts#L70
?

**Comment by T1m1** [comment]
yes, i allready have implemented the feature in plugins group/user

**Comment by timoweiss** [comment]
superb :+1:

**Comment by T1m1** [comment]
@timoweiss: ready for review
